frontend/app/model/ResponsiveImage.scala

```diff
@@ -13,7 +13,7 @@ object ResponsiveImageGenerator {
 }
 
 object ResponsiveImageGroup {
```

Comment:
I feel like overriding `apply` here wasn't great, as we are actually changing the normal signature of apply by returning an `Option[T]` instead of a `T` directly